src/components/ui/Badge/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import React from 'react';
+import {customClassSwitcher} from '~/core'
+
+const COMPONENT_NAME = 'Badge';
+
+interface BadgeProps {
```

Comment:
we should discuss on whether or not we want to export all interfaces. 

i wasnt sure if there were any potential drawbacks from exposing all interfaces for users (alot may be redundant/confusing), but in terms of rad-ui development, it would make things easier, yes.